client/bindings/nodejs/lib/bindings.ts

```diff
@@ -4,7 +4,7 @@
 import type { MessageHandler } from './MessageHandler';
 
 // @ts-ignore: path is set to match runtime transpiled js path
-import addon = require('../../index.node');
+import addon = require('../../build/Release/index.node');
```

Comment:
Just checking, will this always be `Release`? Is it not dependent on how it's being built?

Reply:
Yes, https://github.com/iotaledger/iota.rs/blob/c5b53f446fe523e7e8968d85d3161f47bbae33ce/client/bindings/nodejs/scripts/move-artifact.js#L4

Reply:
Or wait actually no, it will be build in another place, but then moved, so if you run it without the move script, then it's wrong

Reply:
But `npm i` will run it

Reply:
So we good ?

Reply:
I would say so, CI is fine and released package works also, would only be an issue if someone runs some other commands individually